### PR TITLE
Add operations section to k8s notes

### PR DESCRIPTION
**Pull request by lkysow**
* New sidebar for operations
* Move some sections in there
* New sidebar for Troubleshooting -> links to main Consul FAQ/Common errors pages

### CODE REVIEW

File: website/content/docs/k8s/operations/faq.mdx
Comment (ishustava) [inline website/content/docs/k8s/operations/faq.mdx:8]: Do you think this should be a section under Consul's main [FAQ](https://www.consul.io/docs/troubleshoot/faq) page? I think this would make it more discoverable. Or is your thinking to have an FAQ page specific to operations?

File: website/content/docs/k8s/operations/faq.mdx
Comment (ishustava) [inline website/content/docs/k8s/operations/faq.mdx:18]: I think it'd be helpful to link to the upgrade instructions here (https://www.consul.io/docs/k8s/upgrade)?

**Review by ishustava — COMMENTED** [review]
Looking good! I've added some comments, the main one being about how should we treat FAQ page(s). Otherwise, looks good to me!

**Comment by lkysow** [comment]
@ishustava I've added the FAQs to the main page and then linked back to it from within kube->troubleshoot.

I also kept the Upgrade page at top-level to match the consul nav (rather than putting it under operations).

**Review by ishustava — APPROVED** [review]
Awesome, looks great!

**Comment by thisisnotashwin** [comment]
Will merge this in after #9625 gets merged!

File: website/content/docs/troubleshoot/faq.mdx
Comment (ndhanushkodi) [inline website/content/docs/troubleshoot/faq.mdx:22]: If there are backup and restore docs, can we link them here?

**Review by ndhanushkodi — APPROVED** [review]
Looks awesome! Just a comment on whether to include a link for backing up consul data

File: website/content/docs/troubleshoot/faq.mdx
Comment (lkysow) [reply website/content/docs/troubleshoot/faq.mdx:22]: 👍 linked to https://learn.hashicorp.com/tutorials/consul/kubernetes-disaster-recovery

**Pull request merged**
